src/coreclr/vm/eeconfig.h

```diff
@@ -124,6 +124,10 @@ class EEConfig
     }
 #endif // FEATURE_GDBJIT && _DEBUG
 
+#if defined(FEATURE_CACHED_INTERFACE_DISPATCH) && defined(FEATURE_VIRTUAL_STUB_DISPATCH)
```

Comment:
I'm wondering whether we should always have this getter available and have it just return FALSE when it's not possible to enable it.

In some places we only `#if` on `FEATURE_CACHED_INTERFACE_DISPATCH` while here we're doing it on `FEATURE_CACHED_INTERFACE_DISPATCH && FEATURE_VIRTUAL_STUB_DISPATCH` which could lead to a mismatch with some code erroneously trying to call this getter when it's not actually there. Feels error-prone.